Conan.VisualStudio/Services/VcProjectService.cs

```diff
@@ -36,21 +34,48 @@ public bool IsConanProject(Project project)
             return IsCppProject(project) && null != ConanPathHelper.GetNearestConanfilePath(AsVCProject(project).ProjectDirectory);
         }
 
-        public VCProject AsVCProject(Project project)
+        private static IVCProject CreateVCProjectWrapper(Project project)
+        {
+            int version = ConanCompilerVersion();
```

Comment:
I feel like `ConanCompilerVersion` is not the proper name to retrieve the version of VS (More on this on the review comment)

Reply:
I gonna rename it

Reply:
renamed to VisualStudioVersion (property)